core/storage/exploration/gae_models.py

```diff
@@ -389,8 +391,8 @@ class ExpSummaryModel(base_models.BaseModel):
     owner_ids = ndb.StringProperty(indexed=True, repeated=True)
     # The user_ids of users who are allowed to edit this exploration.
     editor_ids = ndb.StringProperty(indexed=True, repeated=True)
-    # The user_ids of users who are allowed to translate this exploration.
-    translator_ids = ndb.StringProperty(indexed=True, repeated=True)
```

Comment:
will this clean the datastore i.e, after this PR whether this translator_ids will get deleted from the expSummary? If not please do it in a similar way you have done above so that while doing cleanup one can delete these values too.

Reply:
Does setting translator_ids to empty count as "cleaning the datastore"? If so, I have done it.

Reply:
Sorry I didn't realize it was the summary model. I will clean it!

Reply:
Done.